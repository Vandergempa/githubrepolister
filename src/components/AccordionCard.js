import React from "react"

import Accordion from 'react-bootstrap/Accordion'
import Card from 'react-bootstrap/Card'

const AccordionCard = (props) => {
  return (
    <Card>
      <Accordion.Toggle as={Card.Header} eventKey={props.eventKey}>
        <strong>{props.title}</strong>
      </Accordion.Toggle>
      <Accordion.Collapse eventKey={props.eventKey}>
        <Card.Body>{props.body}</Card.Body>
      </Accordion.Collapse>
    </Card>
  )
}

export default AccordionCard