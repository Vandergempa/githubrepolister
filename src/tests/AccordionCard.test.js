import React from 'react';
import { shallow } from 'enzyme';
import AccordionCard from '../components/AccordionCard';

test('Should render AccordionCard component correctly', () => {
  let wrapper = shallow(
    <AccordionCard
      eventkey={4}
      title={"Ez"}
      body={"Valami"}
    />
  );
  expect(wrapper).toMatchSnapshot();
});