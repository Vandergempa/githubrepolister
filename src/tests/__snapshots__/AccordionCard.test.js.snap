// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render AccordionCard component correctly 1`] = `
<Card
  body={false}
>
  <ForwardRef
    as={
      Object {
        "$$typeof": Symbol(react.forward_ref),
        "defaultProps": undefined,
        "displayName": "CardHeader",
        "render": [Function],
      }
    }
  >
    <strong>
      Ez
    </strong>
  </ForwardRef>
  <AccordionCollapse>
    <CardBody>
      Valami
    </CardBody>
  </AccordionCollapse>
</Card>
`;
